examples/web/package.json

```
@@ -27,29 +27,26 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
   },
   "devDependencies": {
-    "@babel/core": "^7.12.10",
-    "babel-loader": "^8.2.2",
+    "@babel/core": "^7.21.8",
+    "babel-loader": "^8.3.0",
     "ts-loader": "^6.2.2",
-    "webpack": "^4.44.2",
+    "webpack": "^4.46.0",
     "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "^3.11.0",
+    "webpack-dev-server": "^3.11.3",
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.3",
-    "@opentelemetry/auto-instrumentations-web": "~0.26.0",
-    "@opentelemetry/context-zone": "~1.0.0",
-    "@opentelemetry/core": "~1.0.0",
-    "@opentelemetry/exporter-otlp-http": "~0.26.0",
-    "@opentelemetry/instrumentation": "~0.26.0",
-    "@opentelemetry/instrumentation-document-load": "~0.26.0",
-    "@opentelemetry/instrumentation-user-interaction": "~0.26.0",
-    "@opentelemetry/instrumentation-xml-http-request": "~0.26.0",
-    "@opentelemetry/propagator-b3": "~1.0.0",
-    "@opentelemetry/resources": "~1.0.0",
-    "@opentelemetry/sdk-trace-base": "~1.0.0",
-    "@opentelemetry/sdk-trace-web": "~1.0.0",
-    "@opentelemetry/semantic-conventions": "~1.0.0"
+    "@opentelemetry/api": "^1.4.1",
+    "@opentelemetry/auto-instrumentations-web": "^0.32.2",
+    "@opentelemetry/context-zone": "^1.13.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.39.1",
+    "@opentelemetry/instrumentation": "~0.39.1",
+    "@opentelemetry/instrumentation-document-load": "^0.32.2",
+    "@opentelemetry/instrumentation-user-interaction": "^0.32.3",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.39.1",
+    "@opentelemetry/propagator-b3": "^1.13.0",
+    "@opentelemetry/sdk-trace-web": "^1.13.0",
+    "@opentelemetry/semantic-conventions": "~1.13.0"
```

```suggestion
    "@opentelemetry/semantic-conventions": "^1.13.0"
```
to stay consistent with the line above